### PR TITLE
Adds screen reader only link to download Form 8 PDF, and another screen reader only link to skip iframe

### LINKED ISSUE #354: Accessibility: PDF.js reads form fields and labels in a confusing order
Info here: https://github.com/department-of-veterans-affairs/appeals-pm/issues/552

### PR DESCRIPTION
**Pull request by amprokop**
Adds screen reader only content since our browser PDF viewer doesn't display the content in an accessible way.

Fixes #354

### CODE REVIEW

File: app/assets/stylesheets/_main.scss [hunk 1]
```diff
@@ -27,3 +27,17 @@
     .bar { fill: $color-secondary; }
   }
 }
+
+// This class will place an element off screen without
+// affecting its placement in tab order, thus making it
+// invisible onscreen but read out by screen readers.
+// Screen reader only content is not generally recommended,
+// so only use this to resolve major technical hurdles.
+.off-screen {
```
Comment (shanear) [inline app/assets/stylesheets/_main.scss]: we already have a class that does this. `.usa-sr-only`

Comment (shanear) [inline app/assets/stylesheets/_main.scss]: 🇺🇸

File: spec/feature/confirm_certification_spec.rb [hunk 2]
```diff
@@ -32,4 +32,13 @@
 
     expect(certification.reload.completed_at).to eq(Time.zone.now)
   end
+
+  scenario "Visiting certification page as a sighted user" do
```
Comment (shanear) [inline spec/feature/confirm_certification_spec.rb]: I'm thinking this test should actually click the link, to make sure it downloads the pdf.

Also you can add this stuff into another test that's confirming the certification. Seems a little small to be in its own test.

**Comment by amprokop** [comment]
@shanear — addressed your comments. i added more test logic. seems to me that there's enough weight in the new test to justify breaking it out into its own scenario, but i can move it back if you prefer! mind reviewing the changes?

**Comment by shanear** [comment]
@amprokop agreed, sounds good. 👍

**Pull request merged**